apps/datahub/src/app/record/header-record/header-record.component.html

```diff
@@ -46,7 +46,7 @@
     class="uppercase font-medium tracking-wider"
     translate
   >
-    record.metadata.access
+    record.metadata.download
```

Comment:
The old key should be removed from translations I think

Reply:
Indeed i forgot to run the npm command !
Thanks